src/QrCode.php

```diff
@@ -440,13 +440,14 @@ private function encode()
 
 			$flag = true;
 			while ($flag) {
+				$wordDataValue = isset($this->wordData[$wordCount]) ? $this->wordData[$wordCount] : null;
 				if ($remainingBit > $bufferBit) {
-					$this->wordData[$wordCount] = ((@$this->wordData[$wordCount] << $bufferBit) | $bufferVal);
+					$this->wordData[$wordCount] = (($wordDataValue << $bufferBit) | $bufferVal);
```

Comment:
why bitshift the value anyway when we know it is null? Is this necessary?

Reply:
Good question. I don't work with operators like this, I thought you would know better. Why does the algorithm try to access a non-existent index?